airflow/models/skipmixin.py

```diff
@@ -120,8 +120,13 @@ def skip(
             return
```

Comment:
If my suggestion for relocating the "skipping" logic to the task_runner makes sense, I think that this Mixin will be eventually deleted (you could ignore the changes here, it's from #44925).

Reply:
Irrelevant for now